homeassistant/components/binary_sensor/nest.py

```diff
@@ -105,14 +104,19 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     add_devices(sensors, True)
 
 
-class NestBinarySensor(NestSensor, BinarySensorDevice):
+class NestBinarySensor(NestSensorDevice, BinarySensorDevice):
     """Represents a Nest binary sensor."""
 
     @property
     def is_on(self):
         """Return true if the binary sensor is on."""
         return self._state
 
+    @property
+    def device_class(self):
+        """Return the device class of the binary sensor."""
+        return _VALID_BINARY_SENSOR_TYPES.get(self.variable, None)
```

Comment:
`None` is the default returned value of `dict.get(key)` when the key is missing in the dict.